integration-cli/docker_cli_events_unix_test.go

```diff
@@ -79,7 +79,7 @@ func (s *DockerSuite) TestEventsOOMDisableFalse(c *check.C) {
 }
 
 func (s *DockerSuite) TestEventsOOMDisableTrue(c *check.C) {
-	testRequires(c, DaemonIsLinux, oomControl, memoryLimitSupport, NotArm, swapMemorySupport)
+	testRequires(c, DaemonIsLinux, oomControl, memoryLimitSupport, NotArm, swapMemorySupport, notPpc64le)
```

Comment:
This one seems to be passing still or was it failing as well?

Reply:
Erm, meant the `DisableFalse` one

Reply:
Yeah all three fail. This one is a bit flakier